nixos/modules/services/misc/ethminer.nix

```diff
@@ -0,0 +1,109 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.ethminer;
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.ethminer = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "enable ethminer ether mining.";
+      };
+
+      recheck = mkOption {
+        type = types.int;
+        default = 1000;
+        description = "time to recheck.";
```

Comment:
please specify the unit (seconds, I imagine ?)

Reply:
milliseconds :)